Automattic/vip-go-mu-plugins
# Cleanup redundant wp-parsely files

## Description

We're updating the way wp-parsely-* folders are added after https://github.com/Automattic/vip-go-mu-plugins/pull/3025. Instead of relying on GitHub tags, we are now downloading production releases from WordPress.org, which results in way less contents in this repository.

## Changelog Description

### Cleanup redundant wp-parsely files

We deploy only essential wp-parsely related files now.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Make sure Parse.ly is enabled.
2. Test that different versions work as normal.
